utilities/helperFunctions.js

```diff
@@ -75,6 +77,27 @@ const updateModMessage = (status, channel_id, text, user_id, ts, moderator) => {
     });
     return;
   }
+
+  const randomEmoji = (sentiment) => {
+    let emojis = [];
+    switch(sentiment){
+      case "happy":
+        emojis = emojisList.happy;
+        break;
+      case "medium":
+        emojis = emojisList.medium;
+        break;
+      case "sad":
+        emojis = emojisList.sad;
+        break;
+      default:
```

Comment:
Correct me if I'm wrong, but this means that if a sentiment isn't specified, it just picks a random one?

Reply:
Yeah it picks a random emoji out of the entire JSON file